tests/test_tests.py

```diff
@@ -183,23 +184,24 @@ def test_that_a_process_can_be_serialized_and_deserialized():
 def test_that_the_grep_date_test_correctly_interprets_exit_code_0_and_1(
     test_files, mocker
 ):
+    # 1 is pass, 0 is fail
     tiff_file = test_files["tiff"]
     target = SingleTarget(tiff_file)
     with TemporaryDirectory() as tmp_dir:
         path_0 = Path(tmp_dir, "code_0.txt")
         path_1 = Path(tmp_dir, "code_1.txt")
         path_0.write_text("0")
         path_1.write_text("1")
-        good_outputs = {"std_out": path_1, "std_err": path_1, "exit_code": path_0}
-        bad_outputs = {"std_out": path_0, "std_err": path_0, "exit_code": path_1}
+        zero_outputs = {"std_out": path_1, "std_err": path_1, "exit_code": path_0}
```

Comment:
Just an observation, but it took me a second to realize zero outputs here relates to the exit code and not the number of outputs.

Reply:
I reverted back to the old `pass`- and `fail` outputs variable naming. I think the comment in each test is enough for comprehension, and I agree that this change was confusing.